css/chat.scss

```diff
@@ -0,0 +1,357 @@
+/*
+ * Copyright (c) 2016
+ *
+ * This file is licensed under the Affero General Public License version 3
+ * or later.
+ *
+ * See the COPYING-README file.
+ *
+ */
+
+#chatTabView .comments,
```

Comment:
is the namespace (`#chatTabView` vs. `#commentsTabView`) the only change in this file?

Reply:
Yes, and I have a fix for that, but due to conflicts it can not be pushed until https://github.com/nextcloud/spreed/pull/1309 is merged.